---
layout: default
title: Alessandro Vullo, Software & Services
---

<div class="row">
  <div class="col-md-12 rightBorder">
    <div class="rounded_panel txt_lightgrey">
      <div class="top-rounded-panel grey-bg">
	<h1>Software & Services</h1>
      </div>
      <div class="content-rounded-panel">
	<div>
	  <h2>AVLTree - Perl extension for efficient creation and manipulation of AVL balanced binary trees.</h2>
	  <br/>
	  <p align="justify">
	    A module to provide a simple and fast implementation of <a href="http://www.eternallyconfuzzled.com/tuts/datastructures/jsw_tut_avl.aspx" target="_blank">AVL balanced trees</a>. It uses the Perl XS extension mechanism by providing a tiny wrapper around an efficient C library which does the core of the work.
	    Preliminary benchmarking shows this module one order of magnitude faster than a pure perl implementation.
	  </p>
	  <p align="justify">The nodes of an AVL tree object can hold any kind of item, as long as each one of these has an element which can be used to define a partial order on the set of possible items. This is specified by providing,
	    upon tree construction, a reference to a function for comparing any two of the possible items.
	  </p>
	  <p align="justify">The underlying C library is a reinterpretation of the C library originally developed by Julienne Walker. This library has been adapted for dealing directly with Perl (SV) variables.</p>
	  <p><a href="https://github.com/avullo/AVLTree" target="_blank">GitHub</a> <a href="https://metacpan.org/pod/AVLTree" target="_blank">MetaCPAN</a></p>
	  <hr>
	</div>
	<div>
	  <h2>The Track Hub Registry - A global centralised collection of publicly accessible track hubs</h2>
	  <br/>
	  <p align="justify">
	    The goal of the Track Hub Registry is to allow third parties to advertise <a href="http://genome.ucsc.edu/goldenPath/help/hgTrackHubHelp.html#Intro" target="_blank">track hubs</a>, and to make it easier for researchers around the world to discover and use track hubs containing different types of genomic research data.
	  </p>
	  <p><a href="https://www.trackhubregistry.org" target="_blank">Website</a></p>
	  <hr>
	</div>
	<div>
	  <h2>DISULFIND - Prediction of disulfide bonding state and cysteine connectivity</h2>
	  <br/>
	  <p align="justify">
	    DISULFIND is a server for predicting the disulfide bonding state of cysteines and their disulfide connectivity starting from sequence alone. Optionally, disulfide connectivity can be predicted from sequence and a bonding state assignment given as input. The output is a simple visualization of the assigned bonding state (with confidence degrees) and the most likely connectivity patterns.
	  </p>
	  <p><a href="http://disulfind.disi.unitn.it" target="_blank">Website</a> <a href="https://academic.oup.com/nar/article/34/suppl_2/W177/2505671/DISULFIND-a-disulfide-bonding-state-and-cysteine" target="_blank">Publication</a></p>
	  <hr>
	</div>
	<div>
	  <a name="distill"></a>
	  <h2>Distill@UCD - Protein structure prediction</h2>
	  <br/>
	  <p align="justify">
	    Distill is a suite of servers for the prediction of protein structural features: secondary structure; relative solvent accessibility; contact density; backbone structural motifs; residue contact maps at 6, 8 and 12 Angstrom; coarse protein topology. The servers are based on large-scale ensembles of recursive neural networks and trained on large, up-to-date, non-redundant subsets of the Protein Data Bank. Together with structural feature predictions, Distill includes a server for prediction of C&alpha; traces for short proteins (up to 200 amino acids).
	  </p>
	  <p><a href="http://distill.ucd.ie/distill" target="_blank">Website</a> <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-402" target="_blank">Publication</a></p>
	  <hr>
	</div>
	<div>
	  <h2>CSpritz - Prediction of intrinsic protein disorder</h2>
	  <br/>
	  <p align="justify">
	    CSpritz is a web server for the prediction of intrinsic protein disorder. It is a combination of previous Spritz with two novel orthogonal systems developed by our group (Punch and ESpritz). Punch is based on sequence and structural templates trained with support vector machines. ESpritz is an efficient single sequence method based on bidirectional recursive neural networks. Spritz was extended to filter predictions based on structural homologues. After extensive testing, predictions are combined by averaging their probabilities. The CSpritz website can elaborate single or multiple predictions for either short or long disorder. The server provides a global output page, for download and simultaneous statistics of all predictions. Links are provided to each individual protein where the amino acid sequence and disorder prediction are displayed along with statistics for the individual protein. As a novel feature, CSpritz provides information about structural homologues as well as secondary structure and short functional linear motifs in each disordered segment.
	  </p>
	  <p><a href="http://protein.bio.unipd.it/cspritz" target="_blank">Website</a> <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkr411" target="_blank">Publication</a></p>
	</div>
      </div>
    </div>
  </div>
</div>
