<div id="sbBubble" class="rounded_panel txt_lightgrey">
  <div class="top-rounded-panel grey-bg">
    <a name="sbioinf"></a>
    <h1 id="user-sb">Structural Bioinformatics/Machine Learning</h1>
    <hr>
  </div>
  <div class="content-rounded-panel">
    <div id="disulfindBlurb">
      <h2>DISULFIND - Prediction of disulfide bonding state and cysteine connectivity</h2>
      <br/>
      <p align="justify">
	DISULFIND is a server for predicting the disulfide bonding state of cysteines and their disulfide connectivity starting from sequence alone. Optionally, disulfide connectivity can be predicted from sequence and a bonding state assignment given as input. The output is a simple visualization of the assigned bonding state (with confidence degrees) and the most likely connectivity patterns.
      </p>
      <p><a href="http://disulfind.disi.unitn.it" target="_blank">Website</a> <a href="https://academic.oup.com/nar/article/34/suppl_2/W177/2505671/DISULFIND-a-disulfide-bonding-state-and-cysteine" target="_blank">Publication</a></p>
      <hr>
    </div>
    <div>
      <a name="distill"></a>
      <h2>Distill@UCD - Protein structure prediction</h2>
      <br/>
      <p align="justify">
	Distill is a suite of servers for the prediction of protein structural features: secondary structure; relative solvent accessibility; contact density; backbone structural motifs; residue contact maps at 6, 8 and 12 Angstrom; coarse protein topology. The servers are based on large-scale ensembles of recursive neural networks and trained on large, up-to-date, non-redundant subsets of the Protein Data Bank. Together with structural feature predictions, Distill includes a server for prediction of C&alpha; traces for short proteins (up to 200 amino acids).
      </p>
      <p><a href="http://distill.ucd.ie/distill" target="_blank">Website</a> <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-402" target="_blank">Publication</a></p>
      <hr>
    </div>
    <div>
      <h2>CSpritz - Prediction of intrinsic protein disorder</h2>
      <br/>
      <p align="justify">
	CSpritz is a web server for the prediction of intrinsic protein disorder. The server provides a global output page, for download and simultaneous statistics of all predictions. Links are provided to each individual protein where the amino acid sequence and disorder prediction are displayed along with statistics for the individual protein. As a novel feature, CSpritz provides information about structural homologues as well as secondary structure and short functional linear motifs in each disordered segment.
      </p>
      <p><a href="http://protein.bio.unipd.it/cspritz" target="_blank">Website</a> <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkr411" target="_blank">Publication</a></p>
    </div>
  </div>
</div>
