<div id="bioinfBubble" class="rounded_panel txt_lightgrey">
  <div class="top-rounded-panel grey-bg">
    <a name="bioinf"></a>
    <h1 id="user-bioinf">Genomics/Bioinformatcs</h1>
    <hr>
  </div>
  <div class="content-rounded-panel">
    <div id="ensemblBlurb">
      <h2>Bio::DB::HTS - High-throughput sequencing data handling in Perl</h2>
      <br/>
      <p align="justify">
	A Perl module providing an interface to the <a href="http://www.htslib.org/" target="_blank">HTSlib library</a> for reading/writing high-throughput sequencing data like indexed and unindexed SAM/BAM/CRAM sequence alignment databases, Tabix and variant data VCF/BCF files. It provides support for retrieving information on individual alignments, read pairs, and alignment coverage information across large regions. It also provides callback functionality for calling SNPs and performing other base-by-base functions.
      </p>
      <p><a href="https://github.com/Ensembl/Bio-DB-HTS" target="_blank">GitHub</a> <a href="https://metacpan.org/pod/Bio::DB::HTS" target="_blank">MetaCPAN</a></p>
    </div>
    <hr>
    <div id="ensemblBlurb">
      <h2>Ensembl Core API</h2>
      <br/>
      <p align="justify">
	The goal of the <a href="http://www.ensembl.org" target="_blank">Ensembl project</a> is to automatically <a href="https://www.ensembl.org/info/genome/genebuild/genome_annotation.html" target="_blank">annotate available genomes</a>, integrate these annotations with other available biological data and make all this publicly available via the web. The range of available data includes comparative genomics, variation and regulatory data.
      </p>
      <p align="justify">
	The Ensembl core API provides a level of abstraction over the <a href="https://www.ensembl.org/info/docs/api/core/core_schema.html" target="_blank">Ensembl Core databases</a> and is used by the Ensembl web interface, pipeline, and gene-build systems. To external users, the API may be useful to automate the extraction of particular data, to customize Ensembl to fulfil a particular purpose, or to store additional data in Ensembl.
      </p>
      <p><a href="https://github.com/Ensembl/ensembl" target="_blank">GitHub</a> <a href="https://www.ensembl.org/info/docs/api/index.html" target="_blank">API Documentation</a></p>
    </div>
    <hr>
    <div id="ensemblrestBlurb">
      <h2>Ensembl REST API</h2>
      <br/>
      <p align="justify">
	A Web service to access Ensembl data using Representational State Transfer (REST). The Ensembl REST server enables the easy retrieval of a wide range of Ensembl data by most programming languages, using standard formats such as JSON and FASTA while minimizing client work. It also provides bindings to the popular Ensembl <a href="https://www.ensembl.org/info/docs/tools/vep/index.html" target="_blank">Variant Effect Predictor</a> tool permitting large-scale programmatic variant analysis independent of any specific programming language.
      </p>
      <p><a href="https://github.com/Ensembl/ensembl-rest" target="_blank">GitHub</a> <a href="https://rest.ensembl.org" target="_blank">API Documentation</a></p>
    </div>
    <hr>
    <div id="disulfinderBlurb">
      <h2>Disulfinder</h2>
      <br/>
      <p align="justify">
	Disulfinder is the standalone version of <a href="services.html#disulfind">DISULFIND</a>, a popular server for predicting the disulfide bonding state of cysteines and their disulfide connectivity starting from sequence alone.
      </p>
      <p align="justify">
	The application is available on Ubuntu and Debian.
      </p>
      <p><a href="http://packages.ubuntu.com/search?keywords=disulfinder&searchon=names" target="_blank">Download package</a>
    </div>
  </div>
</div>
